kythe/docs/schema/writing-an-indexer.txt

```diff
@@ -444,7 +444,8 @@ set -o pipefail
 # Read JSON entries from standard in and pass them to the verifier.
 # The entrystream tool turns the JSON into length-delimited protocol buffers,
 # described at http://godoc.org/kythe.io/kythe/go/platform/delimited
-/opt/kythe/tools/entrystream --read_json | /opt/kythe/tools/verifier "$@"
+/opt/kythe/tools/entrystream --read_format=json | \
+/opt/kythe/tools/verifier --nofile_vnames --nocheck_for_singletons "$@"
```

Comment:
I don't think we really want to suggest disabling the singletons check as part of the default example, do we?

Reply:
We should probably update the example so that it passes with this check enabled.  The shortest route is to append `?` to the node in question.

Reply:
done